kusto/kcsb.go

```diff
@@ -11,191 +10,351 @@ import (
 )
 
 type connectionStringBuilder struct {
-	clusterURI string
-	authType   string
-	authParams map[string]interface{}
-	cloudInfo  CloudInfo
+	DataSource                       string `json:"DataSource"`
```

Comment:
The struct is private and the members are public. Let's make it consistent (private vars may be)